system_upgrade.py

```diff
@@ -550,6 +565,10 @@ def run_upgrade(self, extcmds):
         Plymouth.progress(0)
         Plymouth.message(_("Starting system upgrade. This will take a while."))
 
+        if not Plymouth.alive:
```

Comment:
I had to comment out this if clause. It breaks if I boot to plymouth splash and then switch to text console. Maybe timing plays a role as well (whether I switch to text console before or after "Starting system upgrade" line is displayed), I don't know exactly how `.alive` behaves when switched to text, but either way, commenting it out seems to help.